frontend/public/components/project-overview.jsx

```diff
@@ -40,10 +45,12 @@ const ProjectOverviewListItem = ({obj}) => {
     </div>
   </div>;
 
-  return <ListViewItem
-    className="project-overview__item"
+  return <ListView.Item
+    onClick={() => isSelected ? onClick({}) : onClick(obj)}
+    className={className}
     heading={heading}
     additionalInfo={[additionalInfo]}
+    actions={<Icon className="project-overview__item-chevron" type="fa" name="chevron-right" />}
```

Comment:
Wondering if we should just add this with CSS

cc @sg00dwin

Reply:
Yeah, that sounds good to me. I fiddled with that for a bit, but didn't want to spend too much time on it.